srcpkgs/river/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'river'
+pkgname=river
+version=0.1.0
+revision=1
+build_style=zig-build
+configure_args="$(vopt_if xwayland -Dxwayland)"
+hostmakedepends="zig pkg-config wayland-devel scdoc git"
+makedepends="wlroots-devel"
+depends="$(vopt_if xwayland xorg-server-xwayland)"
+short_desc="Dynamic tiling Wayland compositor"
+maintainer="Isaac Freund <mail@isaacfreund.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/riverwm/river"
```

Comment:
What about a `post_install` installing https://github.com/riverwm/river/blob/master/example/init ?

Makes it way easier to test!

Reply:
Yeah that's definitely a good call, done.